site/en/blog/origin-trials/index.md

```diff
@@ -56,16 +56,18 @@ available in Chrome by default. Sites needing extra time to migrate off AppCache
 the [deprecation trial to continue using AppCache](/origintrials/#/view_trial/1776670052997660673) until Chrome 93.
 
 
-## How to register for an origin trial
+## Take part in an origin trial
 
 1. Choose an origin trial from the [list of active trials](https://developers.chrome.com/origintrials/#/trials/active).
-1. Request a token by clicking the **Register** button and filling out the form.
-1. Add the token to your web pages,
-   using one of the following methods:
-   -  As a meta tag in the &lt;head&gt; of each page served:
+1. Request a token (a string of letters and numbers) by clicking the **Register** button and
+filling out the form.
+1. Provide the token on every web page for which you want the trial feature to be enabled, using one
```

Comment:
```suggestion
1. Provide the token to every web page participating in this trial, using one
```

Reply:
The "who" for indicating participation is clear at the start, so it doesn't need to be stated. This modification slightly shortens and clears up your intent.

Reply:
Agree that this has become a bit wordy, but I'm uncomfortable with 'every web page participating'. (It's 'you', the developer, who is participating). Also — nit — the token is not provided 'to' the web page: it's made available on the web page 'to' the browser to enable the feature.

I've changed it to 'Provide the token on every web page for which you want the trial feature to be enabled: ...'